ravenpy/models/emulators.py

```diff
@@ -641,6 +663,160 @@ def ost2raven(self, ops):
         return self.params(*out)
 
 
+class SACSMA(Raven):
+
+    # Details about SAC-SMA:
+    # https://wiki.ewater.org.au/display/SD41/Sacramento+Model+-+SRG
+
+    identifier = "sacsma"
+    templates = tuple((Path(__file__).parent / "raven-sacsma").glob("*.rv?"))
+
+    params = namedtuple(
+        "SACSMAParams", ", ".join(["par_x{:02}".format(i) for i in range(1, 22)])
+    )
+
+    @dataclass
+    class HRU(HRU):
```

Comment:
It doesn't make sense to have a derived class named the same as its parent. Please use a more specialized name for the child class.

Reply:
That's on me, but I'm not sure I agree. The scope is different, so it's another object. We could have defined the original HRU class inside a base Emulator class, and we'd just be subclassing this attribute.

Reply:
I let you two decide as I honestly don't really know what this is doing anyway. :)